README.md

```diff
@@ -1,6 +1,42 @@
 # Calva Power Tools
 
-TBD...
+A VSCode extension for Clojure and ClojureScript development that adds commands for popular Clojure libraries and tools.
+
+## Usage
+
+1. Install the extension from the [VSCode Marketplace](https://marketplace.visualstudio.com/items?itemName=PEZ.calva-power-tools).
+1. Open the command palette with <kbd>ctrl+shift+p</kbd> or <kbd>cmd+shift+p</kbd>.
+1. Type `Clay` to see the available Clay commands.
+
+## Supported Libraries
+
+* [Clay](https://scicloj.github.io/clay/) for literate programming and data visuallization.
+
+## Prerequisites
+
+* [Calva](https://marketplace.visualstudio.com/items?itemName=ms-vscode.clojure-extension-pack) installed
```

Comment:
The extension depends on Calva in the manifest, so it will be installed automatically if it isn't already installed.